Keas.Mvc/Controllers/Api/DocumentController.cs

```diff
@@ -34,6 +33,8 @@ public DocumentsController(ApplicationDbContext context, IDocumentSigningService
         // List all documents for this team
         [HttpGet]
         [ProducesResponseType(typeof(IEnumerable<Document>), StatusCodes.Status200OK)]
+        [Produces(MediaTypeNames.Application.Json)]
+        [Consumes(MediaTypeNames.Application.Json)]
         public async Task<IActionResult> List()
```

Comment:
For actions like this and ones that only take query or path parameters, should we not include a Consumes json attribute?

Reply:
Just saw Scott's comment on slack.